fuse/dl/lightning/pl_module.py

```diff
@@ -134,7 +134,7 @@ def training_epoch_end(self, step_outputs) -> None:
 
     def validation_epoch_end(self, step_outputs) -> None:
         # for the logs to be at each epoch, not each step
-        self.log("step", self.current_epoch, on_epoch=True)
+        self.log("step", float(self.current_epoch), on_epoch=True)
```

Comment:
Why?

Reply:
Solves the following warning:

```
<packages_path>/pytorch_lightning/trainer/connectors/logger_connector/result.py:229: UserWarning: You called `self.log('step', ...)` in your `validation_epoch_end` but the value needs to be floating point. Converting it to torch.float32.
```

Now I double checked that, seems I missed the same fix in the `training_epoch_end` function.
Sure we can ignore it :-)

Reply:
Great, thanks